### PR TITLE
Add links to Russian chat

### PR DESCRIPTION
**Pull request by KnorpelSenf**
There is a new Russian community chat under https://t.me/grammyjs_ru as announced [here](https://t.me/grammyjs_news/20). This PR adds several links to it where it makes sense.

It also adds a link to example updates on the Telegram website, cuz that's just super useful IMO :)

### CODE REVIEW

File: site/docs/resources/faq.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ grammY is a completely free and open-source software, developed by a team of vol
 Its code is available on GitHub.
 
 You're welcome to [join us](https://t.me/grammyjs)!
+(If you speak Russian, you can also join us [here](https://grammyjs_ru)!)
```
Comment (WingLim) [inline site/docs/resources/faq.md:20]: Wrong link.
```suggestion
(If you speak Russian, you can also join us [here](https://t.me/grammyjs_ru)!)
```

**Review by WingLim — CHANGES_REQUESTED** [review]
I will fix it.

File: site/docs/resources/faq.md (hunk 1 above)
Comment (KnorpelSenf) [reply site/docs/resources/faq.md:20]: Oops. Thanks.

**Review by WingLim — APPROVED** [review]
🚀

**Pull request merged**